src/xcode/ENA/ENA/Source/AppDelegate & Globals/ENATaskExecutionDelegate.swift

```diff
@@ -88,6 +90,15 @@ class TaskExecutionHandler: ENATaskExecutionDelegate {
 						}
 					}
 
+					group.enter()
+					DispatchQueue.global().async {
+						Log.info("Trying to fetch family member TestResults...", log: .background)
+						self.executeFetchFamilyMemberTestResults { _ in
+							group.leave()
+							Log.info("Done fetching family member TestResults...", log: .background)
+						}
+					}
+
```

Comment:
Another job for our overloaded background task? Is someone aware of the extra load and the potential side effects? Did we talked with some architect about it?

Reply:
As it's part of the tech spec, I think they are aware of it. There was no separate discussion about it, but I think there will be a separate ticket in the next sprint to optimize the background task. But fetching test result should probably be one of the tasks with a higher priority anyway.